### ISSUE TITLE
not getting past List_activities 

### ISSUE THREAD

**Issue by daytonbrown** [petergardfjall/garminexport#107]
This has worked in the past and downloaded all of my activities.  Now it never gets past list_activities.

![image](https://github.com/petergardfjall/garminexport/assets/5418764/411fd150-8623-4390-92fb-07c967d07baa)

**Comment by whathome** [comment]
Seeing this also.

**Comment by seb2020** [comment]
Same issue for me

**Comment by petergardfjall** [comment]
Would anyone mind trying running through the branch in https://github.com/petergardfjall/garminexport/pull/105?
That seems to do the trick for me.

**Comment by lostasakiwi** [comment]
In a fresh venv running python 3.11.6 on ubuntu 23.10 I did the following....

```
source venv/bin/activate
git clone -b browser-tls-fingerprint-bypass https://github.com/petergardfjall/garminexport.git
pip install ./garminexport/
pip install garminexport[impersonate_browser]
garmin-backup username --backup-dir=/root/garmin-data -f tcx --password=passwordhere
```
Downloaded all of my new garmin tcx files like a champ!

**Comment by Egregius** [comment]
Thanks for this fix! Also working here.

**Comment by petergardfjall** [comment]
I just released `garminexport` `0.5.0`, which I believe should resolve this issue.
It is strongly adviced to install with the `impersonate_browser` extra: `pip install "garminexport[impersonate_browser]"`.

**Comment by mocquin** [comment]
@petergardfjall  FYI, I got it working in a new env with 
`
pip install "garminexport[impersonate_browser]"
garmin-backup username
`
Thanks again
